docs/user-guide/configuration.md

```diff
@@ -518,6 +534,28 @@ experimental_create_proxy = "auto"
     [raise an issue](https://github.com/pyscript/pyscript/issues) with a
     reproducable example, and we'll investigate.
 
+### experimental_ffi_timeout
+
+When bootstrapping a worker, the worker is told to use a cache for round-trip
+operations (for example, `my_object.foo.bar.baz` causes a round-trip to the
```

Comment:
`window.foo.bar.baz` ? ... not just any reference, it's important that caching works **only** for main thread (window) related operations